qacode/core/enums/enum_base.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""package qacode.core.enums.enum_base"""
+
+
+from enum import Enum
+
+
+class EnumBase(Enum):
+    """Enum Class with base methods to be used
+        at inherit classes
+    """
+
+    @classmethod
+    def get_properties(cls):

```

Comment:
Not needed, it's already implemented in `Enum`: https://docs.python.org/3/library/enum.html#iteration

Reply:
reading about this... 🤔

Reply:
will be resolved at #181